config/default.js

```diff
@@ -31,7 +31,9 @@ const conf = {
       "location": process.env.WTP_LOCATION_PATH || 'data.location',
       "url": process.env.WTP_URL_PATH || 'data.url',
       "browserName": process.env.WTP_BROWSER_NAME_PATH || 'data.median.firstView.browser_name',
-      "browserVer": process.env.WTP_BROWSER_VER_PATH || 'data.median.firstView.browser_version'
+      "browserVer": process.env.WTP_BROWSER_VER_PATH || 'data.median.firstView.browser_version',
+      "lcp": process.env.LCP_PATH || "data.median.firstView.largestPaints",
+      "lcpURL": process.env.LCP_PATH || "data.median.firstView.LargestContentfulPaintImageURL"
```

Comment:
You probably want to modify the environment variable name to `LCP_URL_PATH` as currently it's the same as the `lcp` one.